.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://postgres@db/test_track_development
```

Comment:
what's our strategy for distributing env files?  what i've seen elsewhere is committing an `env.dist` file to the repo to serve as a template and provide defaults, then the end user copies that into a .gitignored `.env` file and makes their changes as necessary.